developer_manual/client_apis/OCS/ocs-api-overview.rst

```diff
@@ -77,6 +77,24 @@ This request returns the available metadata of a user. Admin users can see the i
 			</backendCapabilities>
 		</data>
 	</ocs>
+	
+User metadata - Avatar
+-------------
+
+Since: 11.0.2, 12.0.0
+
+This request returns the available avatar of a user. The avatar is always a square so PIXEL_SIZE is both height and width. 
+The version parameter is optional, the last version will be returned if ommited.
+
+.. code::
+
+	GET /avatar/USERID/PIXEL_SIZE?v=VERSION
```

Comment:
I think we should add /index.php/ in front, as not all server have the rewrite to omit index.php.

Reply:
Also this is not an OCS API URL, but a normal URL. So it needs another page in the documentation anyways

Reply:
I just put the data where I would expect it as a developer. Is there something I still need to do here?